plugin/grep.vim

```diff
@@ -1,18 +1,18 @@
 " Grep utility
-if !exists("g:grepprg")
+function! PrepGrep()
   let output = system("git rev-parse --is-inside-work-tree")
   let is_git_repo = v:shell_error == 0
 
   if is_git_repo
-    let g:grepprg="git --no-pager grep -n"
+    let g:grepprg="git --no-pager grep -n "
```

Comment:
To be honest I just noticed that line 12, which used the native grep command, had a space, and I wanted them all to be consistent. Upon review, the makeprg line below adds a space between the grepprg and the args always, so no spaces are necessary. I will update my PR to remove my added spaces as well as the one on line 12 which is not necessary.